lib/Locale/Country/Extra.pm

```diff
@@ -33,6 +33,12 @@ sub code_from_country {
     my ($self, $country) = @_;
 
     my %code_countries = reverse %{ $self->_country_codes };
+
+    unless(exists $code_countries{$country}){
```

Comment:
we can return the value directly if there exists such item. Such case is the most case.

and `$code_countries{undef}` will report an error or warning if there is no such country name. So we can set a default country `''` . Please add a test case for it.
I suggest to change to:
```suggestion
    use List::Utils qw(first);
    return lc $code_countries{$country} if exists $code_countries{$country};
    return $code_countries{(first { $country eq lc $_ } keys %code_countries) // ''};
```